buildkite/table_buildkite_build.go

```diff
@@ -105,7 +105,6 @@ func listBuild(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData)
 			case "=":
 				createdAtMinTime = ts
 				createdAtMaxTime = ts
-				break
```

Comment:
@misraved for changes like this to satisfy linting, did you also test the functionality was still the same?